Composer/packages/extensions/intellisense/src/hooks/useLanguageServer.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { CompletionItem } from 'vscode-languageserver-types';
+import { w3cwebsocket as W3CWebSocket } from 'websocket';
+import {
+  getCompletionRequestMessage,
+  getConfigurationChangedMessage,
+  getDocumentChangedMessage,
+  getInitializeMessage,
+  getTextDocumentOpenedMessage,
+} from '../utils/lspMessagesUtils';
+
+const LANGUAGE_NAME = 'intellisense';
```

Comment:
I don't think UPPER_CASE is the right style for constants in TS.  The style across the codebase seems inconsistent, but this feels like a holdover from some C++ stuff.

Reply:
I tried to be consistent with the rest of the app. Check out lg.ts and lu.ts.

Reply:
Blurg.